ccl/auditloggingccl: TestReducedAuditConfig failed

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/b50984f1-1b6c-4c98-a118-837a02e153fc?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [f6e9d3fd43f7fe1ace06387984cb79bfeb6f8600](https://github.com/cockroachdb/cockroach/commits/f6e9d3fd43f7fe1ace06387984cb79bfeb6f8600):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig3140833117
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig3140833117
--- FAIL: TestReducedAuditConfig (28.31s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-52589

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/f4485e5b-33fb-4cf3-920a-c1a5e9838d9c?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [7df163f07fad86b4043f0ac3690591c97f70bf82](https://github.com/cockroachdb/cockroach/commits/7df163f07fad86b4043f0ac3690591c97f70bf82):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig2168237250
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig2168237250
--- FAIL: TestReducedAuditConfig (33.23s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/7d7e862d-ce28-4e67-98ec-386f4dc6a0f3?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [cd4d33ed5111bf0526b4690f2e783367b02037f2](https://github.com/cockroachdb/cockroach/commits/cd4d33ed5111bf0526b4690f2e783367b02037f2):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig2735366586
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig2735366586
--- FAIL: TestReducedAuditConfig (28.75s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/6fb544b2-0f61-4492-b8b6-3dc751221e38?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [944facb5251aa9448171d0cc0a8ffd6a7c2e7519](https://github.com/cockroachdb/cockroach/commits/944facb5251aa9448171d0cc0a8ffd6a7c2e7519):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig3231287627
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig3231287627
--- FAIL: TestReducedAuditConfig (29.47s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/3c746036-8e56-44e2-8de0-4bb799032708?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [d4372e060619ade859f6f3b07e67c6042dbf5b06](https://github.com/cockroachdb/cockroach/commits/d4372e060619ade859f6f3b07e67c6042dbf5b06):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig4237936192
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig4237936192
--- FAIL: TestReducedAuditConfig (30.28s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/8866c141-ed44-4070-ad0e-871ddc0b0eae?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [b8c2405930718b735bf009c802798b4918b66631](https://github.com/cockroachdb/cockroach/commits/b8c2405930718b735bf009c802798b4918b66631):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig1424734873
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig1424734873
--- FAIL: TestReducedAuditConfig (26.51s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/39ad39a0-dd5e-4bac-83b9-4519fa23cf91?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [afef2d239966541de7611883fbb803f08c0fed92](https://github.com/cockroachdb/cockroach/commits/afef2d239966541de7611883fbb803f08c0fed92):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig445037059
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig445037059
--- FAIL: TestReducedAuditConfig (28.66s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/782a7ca4-1bb8-481d-be54-6cd1364d7f02?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [afef2d239966541de7611883fbb803f08c0fed92](https://github.com/cockroachdb/cockroach/commits/afef2d239966541de7611883fbb803f08c0fed92):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig245079353
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig245079353
--- FAIL: TestReducedAuditConfig (23.04s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/auditloggingccl.TestReducedAuditConfig [failed](https://mesolite.cluster.engflow.com/invocations/default/36ea5e7c-4d9a-4771-8ffb-ef3b2660655d?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2F1ZGl0bG9nZ2luZ2NjbDphdWRpdGxvZ2dpbmdjY2xfdGVzdA==) on master @ [9be69f8b9cc770c7549c39aa8998a24c718f5f2a](https://github.com/cockroachdb/cockroach/commits/9be69f8b9cc770c7549c39aa8998a24c718f5f2a):


```
=== RUN   TestReducedAuditConfig
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestReducedAuditConfig2996916186
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    audit_logging_test.go:401: unexpected number of entries for COMMIT; expected: 1 found: 2
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestReducedAuditConfig2996916186
--- FAIL: TestReducedAuditConfig (24.08s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReducedAuditConfig.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/auditloggingccl/audit_logging_test.go
+++ b/pkg/ccl/auditloggingccl/audit_logging_test.go
@@ -389,7 +389,7 @@ func TestReducedAuditConfig(t *testing.T) {
 			0,
 			math.MaxInt64,
 			10000,
-			regexp.MustCompile(stmt),
+			regexp.MustCompile(`"Statement":"`+stmt),
 			log.WithMarkedSensitiveData,
 		)
 

